os/binfmt/binfmt_execmodule.c

```diff
@@ -81,7 +81,7 @@
 
 #ifdef CONFIG_BINFMT_ENABLE
 
-#ifdef CONFIG_ARMV7M_MPU
+#if defined(CONFIG_ARMV7M_MPU) || defined(CONFIG_ARMV8M_MPU)
```

Comment:
How about making one config for representing MPU?
For example,
```
config ARM_MPU
   bool

config ARMV7M_MPU
  bool "afdaf"
  select ARM_MPU


config ARMV8M_MPU
  bool "afdaf"
  select ARM_MPU
``

Reply:
@jeongchanKim for this I plan to add the following lines to "os/arch/arm/Kconfig"

```
config ARMV8M_MPU
	bool "MPU support"
	default n
	depends on ARCH_HAVE_MPU
	select ARCH_USE_MPU
	---help---
		Build in support for the ARM Cortex-M33 Memory Protection Unit (MPU).
		Check your chip specifications first; not all Cortex-M33 chips support the MPU.

config ARMV8M_MPU_NREGIONS
	int "Number of MPU regions"
	default 8 if ARCH_CORTEXM33
	depends on ARMV8M_MPU
	---help---
		This is the number of protection regions supported by the MPU.
```

Please let me know if this is appropriate?

Reply:
I mean, we'd better have one representative config for enabling MPU.

Reply:
so you suggest to remove the filter and add one common filter like "#ifdef CONFIG_ARM_MPU"?

Reply:
Yes, right. Because, if another arch-MPU is added, it looks not good to add continously with '||'.

Reply:
Perfect, got it, ill make the changes in the Kconfig and the c file.

Reply:
fixed in 19305da